include/extractor/edge_based_graph_factory.hpp

```diff
@@ -84,7 +84,7 @@ class EdgeBasedGraphFactory
     EdgeBasedGraphFactory &operator=(const EdgeBasedGraphFactory &) = delete;
 
     explicit EdgeBasedGraphFactory(std::shared_ptr<util::NodeBasedDynamicGraph> node_based_graph,
-                                   const CompressedEdgeContainer &compressed_edge_container,
+                                   CompressedEdgeContainer &compressed_edge_container,
```

Comment:
I am not entirely sure where the best location would be. Having this done here adds an additional potential for difficult debugging. If possible, I'd love to have this separated from the EdgeBasedGraphFactory.